evalml/pipelines/components/transformers/imputers/simple_imputer.py

```diff
@@ -47,7 +47,9 @@ def fit(self, X, y=None):
             self
         """
         X = infer_feature_types(X)
-        X = _convert_woodwork_types_wrapper(X.to_dataframe())
+
+        # Not using select because we just need column names, not a new dataframe
```

Comment:
This is basically the only place where `_convert_woodwork_types_wrapper` lives. We need to convert natural language to `Categorical` or else the sklearn imputer will error out. 

I'm doing this to preserve the current behavior. Long-term, I don't think it makes sense for simple_imputer and per_column_imputer to impute natural language columns. I will file an issue for that.

Reply:
Ah good call.

@freddyaboulton I think you already filed an issue for this: #1587. Right?